ratis-netty/src/main/java/org/apache/ratis/netty/server/NettyServerStreamRpc.java

```diff
@@ -166,6 +181,11 @@ public void addRaftPeers(Collection<RaftPeer> newPeers) {
     }
   }
 
+  @VisibleForTesting
+  public CompletableFuture<RaftClientReply> submitClientRequestAsync(RaftClientRequest request) throws IOException {
```

Comment:
Let remove this method (and also TestDataStreamWithCluster; see below) since it is not useful to invoke submitClientRequestAsync() from DataStreamServerRpc.

Reply:
@szetszwo Sorry, if we do not submit RaftClientRequest from NettyServerStreamRpc, how can we submit RaftClientRequest to RaftServerImpl when receive start-transaction.

Reply:
This is similar to GrpcService, which implements RaftServerRpc.  We may keep RaftServer internally and invokes RaftServer.submitClientRequestAsync() directly.